src/lib/run_environment.ml

```diff
@@ -28,10 +28,11 @@ module Tool = struct
       [@@@ocaml.warning "-11"]
       type t = [
         | `Bwa of [ `V_0_7_10 ]
-        | `Custom of string * string
+        (* A tool that is manually installed by retrieving source or binary.*)
```

Comment:
Oh, you're commenting on the documentation! Yes, now I do find it confusing as well. So what's the reason for the ADT in these definitions?